app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/survey/SurveyTextAreaQuestionScreen.kt

```diff
@@ -35,24 +31,11 @@ fun SurveyTextAreaQuestionScreen(
         onAnswerChange(answers.map { surveyAnswer -> surveyAnswer.copy(answer = answer, selected = true) })
     }
 
-    TextField(
+    SurveyTextField(
         value = answer,
         onValueChange = { answer = it },
-        shape = RoundedCornerShape(10.dp),
-        placeholder = {
-            SurveyText(
-                text = answers.first().text,
-                fontSize = 17.sp,
-                color = White20
-            )
-        },
-        colors = TextFieldDefaults.textFieldColors(
-            focusedIndicatorColor = Color.Transparent,
-            cursorColor = Color.White,
-            backgroundColor = White20,
-            disabledIndicatorColor = Color.Transparent,
-            unfocusedIndicatorColor = Color.Transparent
-        ),
+        placeholderText = answers.first().text,
```

Comment:
Just in case. Or is it impossible? 🤔 
```suggestion
        placeholderText = answers.firstOrNull()?.text.orEmpty(),
```